aip/0124.md

```diff
@@ -20,9 +20,9 @@ a resource may have a many-to-many relationship with another resource type.
 A resource **must** have at most one canonical parent, and `List` requests
 **must not** require two distinct "parents" to work.
 
-### Multiple one-to-many associations
+### Multiple many-to-one associations
```

Comment:
Should line 14 be changed to "a resource may have a many-to-one relationship" as well?

Reply:
Good point. Done.